source/reference/minio-mc/mc-batch-status.rst

```diff
@@ -43,7 +47,7 @@ The :mc:`mc batch status` command outputs real-time summaries of job events on a
          :class: copyable
 
          mc [GLOBALFLAGS] batch list TARGET           \
-                                     JOBID
+                                     "JOBID"
```

Comment:
`JOBID` (and only one `JOBID`?) seems to be required, but in the Parameters section it's marked as optional.

Reply:
Not required. If not specified, it returns a summary of whatever job is currently running. Added that to the parameter description.